### DISCUSSION TITLE
Buckeye Regional: match 1 and 2

### DISCUSSION THREAD

**Discussion by EddieCanales** [BadRobots1014/BadRobot2022#24] [General]
First match at buckeye regional.

<hr /><em>This discussion was created from the release <a href='https://github.com/BadRobots1014/BadRobot2022/releases/tag/match-buckeye-1'>Buckeye Regional: match 1</a>.</em>

**Comment by EddieCanales** [comment]
* Autonomous shot was made.
* Drive back worked, but didn't succeed because our alliance accidentally pinned us in.
* Manual driving, shooting, gathering worked well. Did not climb.

**Reply by vchen04** [reply]
Did we not climb because we chose not to (or some other game circumstance), or was it because there was not capability for climbing?

**Reply by EddieCanales** [reply]
We decided not to because the physics of it wasn't quite working during testing. We are talking about making some mechanical changes to allow it to hang after lifting itself. Right now if you lift, it will slowly lower after the round is complete and before judge checks if we're off the ground.

**Comment by EddieCanales** [comment]
This was code was used for round 2 as well.

* Shot didn't go in this time.
* Backed out of the tarmac successfully this time because it wasn't pinned in.
* Attempted to climb, but had to pull itself up to get off the ground. After the match ended the robot lowered slowly until touching the ground. Judge check a little too late.
